packages/intl-numberformat/tests/misc.test.ts

```diff
@@ -207,8 +207,30 @@ it('chose compact pattern with rounded number', () => {
     notation: 'compact',
   })
 
-  expect(nf.format(999.995)).toEqual('1.00K')
-  expect(nf.format(999995000)).toEqual('1.00B')
+  expect(nf.format(999.995)).toEqual('1K')
```

Comment:
hmm why did this change?

Reply:
For the new logic of the SetNumberFormatDigitOptions function

Reply:
This has been fixed, my mistake.